### ISSUE TITLE
request: add manual/auto/stack mode

### ISSUE THREAD

**Issue by pixelartgirl** [mapeditor/tiled#3705]
Hi

add manual/auto/stack mode for tiles like aseprite

https://www.aseprite.org/docs/tilemap/

**Comment by bjorn** [comment]
These "Draw Pixels" modes refer to the way modifications of the pixels of a tile map layer affect the tile map and its tileset. This does not apply to Tiled, since in Tiled you can't modify the tile images. In Tiled, the only available mode is "Draw Tiles" and there are no plans for adding pixel painting to Tiled.

**Comment by pixelartgirl** [comment]
HAHA very funny, a tilemap tool that DONT let you draw tiles

what a world we live in :))))))))))))))

**Comment by pixelartgirl** [comment]
So yoiu can't make a tilemap using Tiled because you cant edit images and can't make tilesets
So there is another tool REQUIRED for making the tilemap. Making a tileset is a part of making the tilemap...

That doesnet sound very logical to me.
A tilemap tool should be able to produce a tilemap on its own, including the tileset.

**Comment by pixelartgirl** [comment]
So in other words: people who DONT have a tool to make an image for the tileset, can't make a tilemap using your tool.
This is the most unlogical thing I have ever heard. Making tileset is an essential part of your tool and you don't implement it...

It's like making a painting without a brush :)

**Comment by bjorn** [comment]
> A tilemap tool should be able to produce a tilemap on its own, including the tileset.

In general I would agree. The first tile map editor I wrote, around 25 years ago now, covered both tile editing, tile map editing and even palette editing on a single screen:

![image](https://github.com/mapeditor/tiled/assets/531764/4d3a0f47-c64e-40f2-9fab-b7943ae52a59)

However, there is now very good image editing software like GIMP, Krita and Aseprite, so when I started on a new tile map editor I decided to focus on just that part. Many people use Tiled alongside their image editor, or with tilesets drawn by others.

Of course, a built-in image editor could suit some users, but it could never match a dedicated tool like Aseprite. I would hence suggest to use such tools alongside each other. To make this easier, Tiled watches your tileset images for changes and can automatically reload them when you save them in the image editor.
